steps/07-stage.sh

```diff
@@ -19,33 +21,33 @@ sed "s/#VERSION#/${VERSION:-0.0.0.0}/" <"$PATCHES/PDFiumConfig.cmake" >"$STAGING
 
 cp "$SOURCE/LICENSES" "$STAGING/LICENSE"
 cp "$BUILD/args.gn" "$STAGING"
-cp -R "$SOURCE/public" "$STAGING/include"
+cp -R "$SOURCE/public" -T "$STAGING/include"
 rm -f "$STAGING/include/DEPS"
 rm -f "$STAGING/include/README"
 rm -f "$STAGING/include/PRESUBMIT.py"
 
 case "$OS" in
   android|linux)
-    mv "$BUILD/libpdfium.so" "$STAGING_LIB"
+    cp "$BUILD/libpdfium.so" "$STAGING_LIB"
```

Comment:
note: the `mv` -> `cp` changes are unrelated; I just did this for local testing so a single step can be run multiple times.